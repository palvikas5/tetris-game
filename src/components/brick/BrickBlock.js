import React from "react";

const Brick = props => {
  const classes = ["brick"];
  classes.push(`brick-${props.row}-${props.col}`);
  return (
    <div
      className={classes.join(" ")}
      style={{ backgroundColor: props.bgColor }}
    />
  );
};

export default Brick;
