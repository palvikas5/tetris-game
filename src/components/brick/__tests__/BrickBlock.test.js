import React from "react";
import { render } from "@testing-library/react";
import BrickBlock from "../BrickBlock";

describe("<BrickBlock />", () => {
  it("should render successfully", () => {
    const { container } = render(<BrickBlock row={2} col={1} />);
    expect(container).toMatchSnapshot();
  });
});
