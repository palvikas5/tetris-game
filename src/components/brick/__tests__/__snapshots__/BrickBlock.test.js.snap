// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BrickBlock /> should render successfully 1`] = `
<div>
  <div
    class="brick brick-2-1"
  />
</div>
`;
